widgets/pie/options.go

```diff
@@ -0,0 +1,50 @@
+package pie
+
+import (
+	"github.com/mum4k/termdash/cell"
+)
+
+// Option defines a function that sets a specific option for the Pie widget.
+type Option interface {
+	// set sets the provided option.
+	set(*options)
+}
+
+// option implements Option.
+type option func(*options)
+
+// set implements Option.set.
+func (o option) set(opts *options) {
+	o(opts)
+}
+
+// options stores the provided options.
+type options struct {
+	colors []cell.Color
+}
+
+// validates the provided options
+// at the moment no validation is performed cause options are not required
+func (o *options) validate() error {
+	return nil
+}
+
+// newOptions creates a new options instance.
+func newOptions() *options {
+	return &options{
+		colors: DefaultColors,
```

Comment:
Did you want to add an option to set custom colors?

Example of an option:

https://github.com/mum4k/termdash/blob/102df20a882547528c83047688595acd2a82c7ef/widgets/button/options.go#L103-L108